lib/modules/forgot_password/forgot_password_module.dart

```diff
@@ -13,6 +13,8 @@ import 'package:survey/core/viper/module.dart';
 import 'package:survey/gen/assets.gen.dart';
 import 'package:survey/modules/screen.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:survey/repositories/auth_repository.dart';
+import 'package:survey/services/locator/locator_service.dart';
```

Comment:
wondering why are we just started to import this in this PR but there is no usage? 🤔

Reply:
@sleepylee it's used in [here](https://github.com/markgravity/flutter-survey/blob/a36d21e16a0a73cba713e5ae6162924288902e67/lib/modules/forgot_password/forgot_password_interactor.dart)